### PR TITLE
create kube proxy config customization tutorial

### PR DESCRIPTION
**Pull request by scraly**

### CODE REVIEW

File: pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+            "tcpFinTimeout":"PT0S",
+            "tcpTimeout":"PT0S",
+            "udpTimeout":"PT0S"
+         }
+      }
+   }
+}
+```
+
+Both kubeProxyMode and customization.* can be modified on cluster reset with the same payload used for creation.
```
Comment (OVH-Team-Guides) [inline pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md]: Hello @scraly, is "customization.*" a typo ?

File: pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+	}
+}
+```
+
+> [!primary]
+>
+> Updating customization.kubeProxy will trigger following actions:
+>
+> - apply updated configMap
+> - then rollout restart kube-proxy so it uses the new configuration
```
Comment (OVH-Team-Guides) [inline pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md:237]: Should it be:

- rollout/restart
- rollout
- restart
?

File: pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+#### The API Explorer
+
+To simplify things, we are using the [API Explorer](https://api.ovh.com/) which allows to explore, learn and interact with the API in an interactive way.
+
+Log in to the API Explorer using your OVHcloud NIC handle.
+
+![Log in to the API Explorer](images/kubernetes-quickstart-api-ovh-com-001.png){.thumbnail}
+
+If you go to the [Kubernetes section](https://api.ovh.com/console/#/kube) of the API Explorer, you will see the available endpoints:
```
Comment (OVH-Team-Guides) [inline pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md]: Are you sure about this link ? It doesn't focus on any part of api.ovh.com

**Comment by OVH-Team-Guides** [comment]
Could you please crop this image: kubernetes-quickstart-api-ovh-com-001.png
So that it's not containing the address bar anymore.

File: pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md (hunk 1 above)
Comment (scraly) [reply pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md]: hi, no it's not a typo, I will add backtips :)

File: pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md (hunk 2 above)
Comment (scraly) [reply pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md:237]: "kubectl rollout restart" is a command :)

File: pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md (hunk 3 above)
Comment (scraly) [reply pages/platform/kubernetes-k8s/customizing-kubeproxy/guide.en-gb.md]: thanks, it's an old endpoint that I need to fix in other tutos :)

**Comment by scraly** [comment]
Thanks for your remarks, I fixed everything :)

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @scraly

**Pull request merged**